output/eu_crude/analysis_simplified.py

```diff
@@ -0,0 +1,61 @@
+import sqlite3 as lite
+import sys
+import matplotlib as plt
+from pyne import nucname
+
+con = lite.connect('eu_crude.sqlite')
+
+with con:
+
```

Comment:
Why are both analysis.py and analysis_simplified.py in this repository? They seem to be slightly variant copies of one another.